src/main/resources/BridgeServer2.conf

```diff
@@ -199,8 +177,13 @@ gbf.api.key = dummy-value
 query.param.allowlist = type,appId,studyId,IdFilter,assignmentFilter,externalId,identifier,ownerId,newIdentifier,name,notes,tags,includeDeleted,physical,format,summary,startTime,endTime,pageSize,offsetKey,offsetBy,tag,category,minRevision,maxRevision,queryParam,createAccount,createdOnStart,createdOnEnd,consents,scheduledOnStart,scheduledOnEnd,startDate,endDate,deleteReauthToken,until,daysAhead,minimumPerSchedule,mostRecent,mostrecent,published,newSchemaRev,synchronous,redrive
 
 # Participant File S3 bucket name
-dev.participant-file.bucket = org-sagebridge-participantfile-dev
-uat.participant-file.bucket = org-sagebridge-participantfile-uat
-prod.participant-file.bucket = org-sagebridge-participantfile-prod
-# Set in your ~/BridgeServe2.conf file
-# local.participant-file.bucket = org-sagebridge-participantfile-prod-local-[bridge-user]
+participant-file.bucket = org-sagebridge-participantfile-${bucket.suffix}
+
+# Public documents folder
+docs.bucket = docs${host.postfix}
+
+# Note however that the CloudFormation-created buckets are not this straight-forward, and
+# will have to be special-cased when added. 
+health.data.bucket.raw = bridgeserver2-${bucket.suffix}-rawhealthdatabucket
+health.data.bucket.processed = bridgeserver2-${bucket.suffix}-processedhealthdatabucket
```

Comment:
Don't worry about these for now. These *are* created by Infra and are being wired up automatically. (We can migrate these off Infra and onto the S3Initializer later.)